src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,54 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.UserDTO;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("api/v1/users")
 public class UserController {
+
+    private static final List<UserDTO> USERS = Arrays.asList(
+            new UserDTO(1,"Piotr", "1",59403),
+            new UserDTO(2,"Admin", "1",96997),
+            new UserDTO(3,"User", "0",31251)
+    );
+
+    @GetMapping
+    public ResponseEntity<List<UserDTO>> getAllUsers(){
+        return ResponseEntity.ok(USERS);
+    }
+
+    @GetMapping(value = "{userId}")
+    public ResponseEntity<UserDTO> getUserById(@PathVariable long userId){
+
+        List<UserDTO> usersWithMatchingId= USERS.stream().filter(userDTO -> userDTO.getId() == userId)
```

Comment:
Nie jest to potrzebne na tym etapie, zwróć zwykły obiekt utworzony ręcznie

Reply:
ok